### PR TITLE
Access denied

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
:cl:
 * rscadd: New rare noise for not having access to airlocks, or always plays when it's april fools.

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1368,7 +1368,10 @@ About the new airlock wires panel:
 				return
 
 			if (locate(/mob/living) in T)
-				playsound(src, 'sound/machines/buzz-two.ogg', 50, 0)
+				if((Holiday = APRIL_FOOLS_DAY) || prob(1))
```
Comment (Zth--) [inline code/game/machinery/doors/airlock.dm]: sneaky equal :hocho:

**Comment by utilitas-** [comment]
You do realize it only says "access", right?

**Comment by boy2mantwicethefam** [comment]
What?

**Comment by utilitas-** [comment]
Nevermind, MPC is a piece of shit that skips half the file when looping back around. It's all there.
_Sorry, Mr. Freeman, I got explicit orders not to let you through without your hazard suit on._

**Comment by Kurfursten** [comment]
Since you can bump a door like 5 times a second you can pretty easily force out this voice line by holding an arrow into a door for 20 seconds.

That might even be a conservative estimate

**Comment by Irradiation** [comment]
The guy coding this is also the same guy who made a macro to spamfreeze Give-Item people. You're being too conservative Kurfurst.

**Comment by Exxion** [comment]
Unless it's been fixed, you bump the door like 30 times a second if you push against in in a mech

**Comment by SonixApache** [comment]
You can bump it even faster by walking into it while clickspamming it.

Also, didn't HL1 doors have their own 'access denied' sound? Why not use that?

**Comment by utilitas-** [comment]
How about this: Plays only for science/research/toxins/research outpost doors and access to the SME
Both sounds, the one that plays with the retinal scanner and the one for normal doors could be used.

**Comment by SkowronX** [comment]
this is going to get annoying fast

**Pull request closed without merging**